packages/replay-next/components/elements/ElementsListData.ts

```diff
@@ -214,8 +226,7 @@ export class ElementsListData extends GenericListData<Item> {
         const indentation = "  ".repeat(depth);
         const nodeName = element.node.nodeName.toLowerCase();
         rows.push(
-          `${indentation} ${id}:${nodeName}${
-            childrenCanBeRendered ? "" : " *"
+          `${indentation} ${id}:${nodeName}${childrenCanBeRendered ? "" : " *"
```

Comment:
Your editor doesn't seem to be properly configured to respect the Trunk/Prettier formatting. This change is a regression which is why [Lint is failing on your PR](https://github.com/replayio/devtools/actions/runs/6938390329/job/18874032684?pr=9916).